mev_inspect/classifiers/specs/uniswap.py

```diff
@@ -118,7 +118,7 @@
 )
 
 
-CLASSIFIER_SPECS = [
+UNISWAP_CLASSIFIER_SPECS = [
```

Comment:
AAVE_SPEC, ERC20_SPEC, and WETH_SPEC should be in their own files instead of the uniswap one

Reply:
Done.  ERC20 must be the first in the ALL_CLASSIFIER_SPECS list or arbs won't be caught in testing.